docs/configuration/cron_jobs.rst

```diff
@@ -112,11 +112,11 @@ Process Email queue cron job
 
 .. vale on
 
-If the system configuration is queueing Emails, a cron job processes them.
+If the system configuration is queueing Emails, a cron job processes them. If you plan to run `messenger:consume` on a cron you should include, at minimum one of these parameters `--memory-limit`, `--limit` (number of emails), `--time-limit` otherwise, this will start up a long-lived process.
```

Comment:
```suggestion
If the system configuration is queueing Emails, a cron job processes them. If you plan to run `messenger:consume` on a cron you should include, at minimum, one of these parameters `--memory-limit`, `--limit` - the number of emails - or `--time-limit`, otherwise this starts up a long-lived process.
```

Reply:
Would it be helpful to show how you'd use this in real life with an example command, so people understand how to specify the numbers?

Reply:
> Would it be helpful to show how you'd use this in real life with an example command, so people understand how to specify the numbers?

I did update the [command just below](https://github.com/mautic/user-documentation/pull/358/files#diff-84e2392b487553c90d7101e25e7ace972ea4af522f73c2d5fdbab3b191187944R119). Maybe there should be more?

Reply:
That's great, I think you updated it as I wrote the comment!